### PR TITLE
systemd script

### PR DESCRIPTION
**Pull request by uryossa**

### CODE REVIEW

File: examples/openfortivpn.service [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description = Forti VPN for default
+
+[Service]
+ExecStart = /usr/bin/openfortivpn --use-syslog -c /etc/openfortivpn/config
```
Comment (adrienverge) [inline examples/openfortivpn.service]: Can you remove `--use-syslog`? It is not the default and most systems use Systemd's journal.

File: examples/openfortivpn.service [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description = Forti VPN for default
```
Comment (adrienverge) [inline examples/openfortivpn.service]: It's not `Forti VPN`, it's `openfortivpn` ;-)

File: examples/openfortivpn.service [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[Unit]
```
Comment (adrienverge) [inline examples/openfortivpn.service]:
```
Requires = network-online.target 
```
would be useful in this section, what do you think?

**Comment by uryossa** [comment]
I'm sorry.
I don't have enough time for understanding english and learning github.

**Pull request closed without merging**